crates/proof-of-sql/src/sql/proof/first_round_builder.rs

```diff
@@ -8,19 +8,28 @@ pub struct FirstRoundBuilder {
     num_post_result_challenges: usize,
     /// The extra one evaluation lengths used in the proof.
     one_evaluation_lengths: Vec<usize>,
-}
-
-impl Default for FirstRoundBuilder {
-    fn default() -> Self {
-        Self::new()
-    }
+    /// The max of all `range_lengths` used in sumcheck.
```

Comment:
Sorry let's just make it `// The `range_length` used in sumcheck which is max of all possible ones.`